docs/features/hooks/hook-reference.mdx

```diff
@@ -0,0 +1,457 @@
+---
+title: "Hook Reference"
```

Comment:
You used a sample with PreCompact in he examples page but it is not referenced here. Make sure to either add the reference to PreCompact or remove the example until we get the feature here.